Index: bazel/toplevel.WORKSPACE
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bazel/toplevel.WORKSPACE b/bazel/toplevel.WORKSPACE
--- a/bazel/toplevel.WORKSPACE	(revision f8bfc4a722c51720c45b27b3e3a3354295afb450)
+++ b/bazel/toplevel.WORKSPACE	(date 1628091149406)
@@ -10,15 +10,6 @@
   "@native_toolchain//:cc-toolchain-x64_windows-clang-cl",
 )

-local_repository(
-    name = "blaze",
-    path = "tools/vendor/google3/blaze",
-)
-
-load("@blaze//:binds.bzl", "blaze_binds")
-
-blaze_binds()
-
 local_repository(
     name = "io_bazel_rules_kotlin",
     path = "tools/external/bazelbuild-rules-kotlin",
Index: build-system/integration-test/integration-tests.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build-system/integration-test/integration-tests.gradle b/build-system/integration-test/integration-tests.gradle
--- a/build-system/integration-test/integration-tests.gradle	(revision f8bfc4a722c51720c45b27b3e3a3354295afb450)
+++ b/build-system/integration-test/integration-tests.gradle	(date 1628091146990)
@@ -81,6 +81,7 @@
     outputs.upToDateWhen { false }

     maxParallelForks = Runtime.runtime.availableProcessors() / 4
+    maxParallelForks = max(1, maxParallelForks)

     useJUnit {
         String includedCategories = providerFactory.systemProperty('test.includeCategories').forUseAtConfigurationTime().orNull
